crates/astria-composer/src/config/searcher.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    net::{
+        AddrParseError,
+        SocketAddr,
+    },
+    str::FromStr,
+};
+
+use figment::{
+    providers::{
+        Env,
+        Serialized,
+    },
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+use thiserror::Error;
+
+use super::cli;
+use crate::types::rollup::ChainId;
+
+const DEFAULT_SEQUENCER_URL: &str = "127.0.0.1:1317";
+const DEFAULT_API_URL: &str = "127.0.0.1:8080";
+const DEFAULT_CHAIN_ID: &str = "ethereum";
+const DEFAULT_EXECUTION_RPC_URL: &str = "127.0.0.1:50051";
```

Comment:
why do we need this? what are we submitting to the gRPC execution server?

Reply:
will be using this to initialize an [`ethers::Provider`websocket connection](https://www.gakonst.com/ethers-rs/providers/ws.html) (the actual value will probably change, this is just a placeholder copied over from `conductor`)

Reply:
Ok should definitely not be 50051 but instead 8546, can we rename to something like ETH_WS_URL?

Port 50051 is the port for driving execution through our gRPC [Execution API](https://github.com/astriaorg/astria/blob/main/crates/astria-proto/proto/astria/execution/v1/execution.proto)